src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const promise1 = new Promise(function (resolve, reject) {
+  logo.addEventListener('click', () => resolve());
+});
+
+promise1
+  .then((mes) => {
+    const messageSuccess = document.createElement('div');
+
+    messageSuccess.classList.add('message');
+    messageSuccess.textContent = 'Promise was resolved!';
+    document.body.appendChild(messageSuccess);
+  })
+  .catch((mes) => {
```

Comment:
The variable `mes` is not used in the `catch` handler. Consider removing it or using a more descriptive name if needed for future use.